src/Compatibility/Microsoft.DotNet.Compatibility/ValidatePackage.cs

```diff
@@ -1,14 +1,14 @@
 ﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+#nullable enable
```

Comment:
Why not just enable nullability on the whole project?

Reply:
Because the project imports tons of files from a Common folder that aren't yet marked as nullable and I didn't want to bloat the PR by marking those as nullable disable. Given that I will submit a follow-up PR, I will probably do that now.